react/Button/Button.less

```diff
@@ -111,3 +111,47 @@
 .fullWidth {
   width: 100%;
 }
+
+.group {
+  display: flex;
+  width: 100%;
+
+  .root {
+    overflow: visible;
```

Comment:
This might created more unexpected results, a button truncates overflow, this is explicitly negating that which could be an undesired side-effect of putting wrapping a `<Button />` in a `<ButtonGroup />`.  You can see below that with this property a button's label can awkwardly extend into the padding:
![screen shot 2017-09-19 at 10 12 39 am](https://user-images.githubusercontent.com/912060/30570241-57f427a6-9d23-11e7-8ecb-6a8d7c8694f3.png)